es/customization/introduction.md

```diff
@@ -28,67 +28,81 @@ Este servicio tiene como objetivo poder ofrecer todos los contenidos dinámicos
 ```
+
 De todos los idiomas que actualmente tiene Consul definidos (`available_locales`) en `config/application.rb` el único que no está en la lista anterior y por lo tanto no se ofrece servicio de traducción es el valenciano `["val"]`.
 
-#### Costes
+### Costes
+
 El servicio de traducción utilizado tiene los [precios](https://azure.microsoft.com/es-es/pricing/details/cognitive-services/translator/) más competitivos del mercado.
 El precio por cada 1 Millón de caracteres traducidos asciende a 10 $ y sin ningún tipo de coste fijo al mes.
 
 Aunque se han tomado medidas técnicas para evitar un mal uso de este servicio, recomendamos la creación de Alertas que ofrece Azure para que un Administrador pueda ser notificado en el caso de detectar un uso fuera de lo común del servicio. Este servicio tiene un coste de 0,10 $ al mes.
 
 Para crear una Alerta en Azure debemos seguir los siguientes pasos:
+
 1. Inicie sesión en **Azure Portal**.
 1. Accede al servicio **Traductor** creado anteriormente.
 1. Accede en el menu lateral a **Supervisión > Alertas**:
-  1. Accedemos a **Crear regla de alertas**
-  1. En **Selección de una señal** seleccionamos `Text Characters Translated`
-  1. Una vez seleccionada debemos definir la lógica de la Alerta para que se ajuste a nuestras necesidades. Ej: Rellene el campo "Operador" con el valor "Mayor que", rellene el campo "Tipo de Agregación" con el valor "Total" y por último rellene el campo "Valor del umbral" por el número de caracteres que consideramos que deben traducirse antes de ser notificados. En esta sección también se puede configurar el periodo de tiempo y la frecuencia de evaluación.
-  1. Para poder ser notificados tenemos que crear un **Grupo de Acciones** y asociarla a esta Alerta que estamos creando. Para ello accedemos al botón de **Crear** y rellenamos el formulario. Como se puede observar hay diferentes tipos de acciones, debemos seleccionar **Correo electrónico/SMS/Insertar/Voz** y configurar la opción que consideremos conveniente según nuestras necesidades.
-  1. Una vez creado este grupo de acciones, ya queda directamente asociado a la regla que estamos creando.
-  1. Por último ya solo queda añadir un nombre y clicar sobre el botón **Revisar y crear**
-
-#### Añadir un nuevo servicio de traducción
+
```

Comment:
I think that adding this line isn't necessary to follow the "Lists should be surrounded by blank lines" rule, and it makes it harder to read the list.

Reply:
I think this line was added in commit "MD022 Headers should be surrounded by blank lines".  

Maybe it would make sense to keep the line blank, right? 🤔

Reply:
The diff might be a bit confusing; I'm trying to comment on the blank line between these two:

```
1. Accede en el menu lateral a **Supervisión > Alertas**:

   1. Accedemos a **Crear regla de alertas**
```